File: SortingAlgo/bubbleSort.c
#include<stdio.h>

void Print(int *Ar, int size){
        
    for (int k=0; k<size; k++){
        printf("%d ", Ar[k]);
    }
    
}

void BubbleSort(int *Ar, int size){
    int swap;
    for(int i=0; i<size-1; i++){
        for(int j=0; j<size-1-i; j++){
            if(Ar[j]>Ar[j+1]){
                swap=Ar[j];
                Ar[j]=Ar[j+1];
                Ar[j+1]=swap;
            }
        }
    }
    
}

int main(){

    int Ar[] = {10, 7, 23, 13 , 22, 53};
    int size = sizeof(Ar)/sizeof(int);

    printf("Before sort \n");
    
    Print(Ar, size);
    
    printf("\nAfter sort \n");

    BubbleSort(Ar, size);

    Print(Ar, size);
}

File: SortingAlgo/bubbleSort.py
Array = [3, 1, 9, 7, 3, 1, 5, 6, 2]

def bubbleSort(Arr):
    for i in range(len(Arr)-1):
        for j in range(len(Arr)-1):
            if Arr[j]>Arr[j+1]:
                Arr[j], Arr[j+1] = Arr[j+1], Arr[j]



print(Array)
bubbleSort(Array)
print(Array)
